docs/developers.md

```diff
@@ -30,23 +41,33 @@ $ make manifests
 /Users/nitish_malhotra/bin/controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=chart/orkestra/crds
 ```
 
-- Build the docker image and push it to (your own) docker registry
+Build the docker image and push it to (your own) docker registry
 
 ```shell
 docker build . -t <your-registry>/orkestra:<your-tag>
```

Comment:
This can be done through makefile and we should consider updating the makefile to support this rather than giving the explicit commands here 😃 @mahalrs  If you wanted to, you could make that change in this PR as well

Reply:
Sure, will do this in the morning.

Reply:
Lets get this in and then you can add a doc update to #294